Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -2011,6 +2020,35 @@ def cybereason_process_attack_tree_command(client: Client, args: dict):
         outputs=outputs)
 
 
+def update_malop_investigation_status_command(client: Client, args: dict):
+    malop_guid = str(args.get('malopGuid'))
+    investigation_status = str(args.get('investigationStatus'))
```

Comment:
```suggestion
    malop_guid = args.get('malopGuid')
    investigation_status = args.get('investigationStatus')
```
No need to use the str() method, as it is already a string

Reply:
If I don't tag it as string, linting will fail. All other commands also do it like that.